tests/core/test_typed_transactions.py

```diff
@@ -353,6 +422,8 @@
     "blob-no-access-list",
     "blob-int-values-and-access-list",
     "blob-no-explicit-type",
+    "sc-standard-case",
+    "sc-and-al",
 ]
```

Comment:
Nice tests!
- It would be good if we could find at least one test with an expected hash from a trusted source like the tests repo or something similar. I'll see if I can find something here too.
- I'd like to see one more test case with more than one authorizationList too.